### PR TITLE
Add version subcommand

### PR DESCRIPTION
**Pull request by pradeepnnv**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 /.popplerc
 /popple
+popple
```
Comment (connorkuehl) [inline .gitignore:3]: Does the line above not ignore the executable?

File: version.go [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package main
+
+// Indicates Popple version
+const version = "1.1.0"
```
Comment (connorkuehl) [inline version.go:4]: Please make it uppercase: `const Version = "1.1.0"`

File: popple.go [hunk 3]
```diff
@@ -19,6 +19,12 @@ const DEFAULT_WORKERS uint = 4
 const DEFAULT_JOBS uint = 128
 
 func main() {
+
+	if len(os.Args) >= 2 && (os.Args[1] == "version") {
+		fmt.Printf("Popple version : %s", version)
+		os.Exit(0)
+	}
+
```
Comment (connorkuehl) [inline popple.go:27]: I appreciate where you're going with this, but Popple isn't an interactive command line tool so I don't think we should add a switch like this. Please remove this.

**Review by connorkuehl — CHANGES_REQUESTED** [review]
The command itself looks good but it is not currently reachable during normal operation.

You need to add an entry to the `cmds` array in `func doWork` in `job.go` so that during message parsing, Popple can recognize the `version` command and execute the code you wrote. Something like this:

```diff
diff --git a/job.go b/job.go
index fcbaddd..a80bd79 100644
--- a/job.go
+++ b/job.go
@@ -28,6 +28,7 @@ func doWork(job *Job, db *gorm.DB) {
                {"help", SendHelp, false},
                {"karma", CheckKarma, true},
                {"top", Top, false},
+               {"version", SendVersion, false},
        }

        msg := job.Message.ContentWithMentionsReplaced()
```

**Comment by connorkuehl** [comment]
Would you also be willing to document this command in the `README.md` with the other commands? Thanks.

**Comment by pradeepnnv** [comment]
@connorkuehl , thanks for the feedback. i've made updates. Please review.

**Comment by connorkuehl** [comment]
Merged as 4137d2f87f4b0b8da12671a2044388290dba8d9f.

Thanks!

**Pull request closed without merging**